api/src/queries/survey/survey-occurrence-queries.ts

```diff
@@ -0,0 +1,54 @@
+import { SQL, SQLStatement } from 'sql-template-strings';
+import { getLogger } from '../../utils/logger';
+
+const defaultLog = getLogger('queries/survey/survey-occurrence-queries');
+
+/**
+ * SQL query to insert a survey attachment row.
+ *
+ * @param fileName
+ * @param fileSize
+ * @param projectId
+ * @param surveyId
```

Comment:
these arent lined up with the actual params of this function

Reply:
Copy/paste stuff, fixed